label_inspector/analysis/grapheme_analysis.py

```diff
@@ -11,6 +11,10 @@
     from .label_analysis import LabelAnalysis
 
 
+def v2num(version: str) -> int:
+    return int(version.replace('.', ''))
```

Comment:
there was a version 1.0.1 (not sure if reflected in files?). maybe better solution would be `(int(n) for n in version.split('.'))`?